auth/shared-signal-receiver/handlers/account-purged-handler.ts

```diff
@@ -19,6 +20,18 @@ export const handleAccountPurgedRequest = async (
     ].subject.uri;
 
   const correlationId = accountPurgedRequest.jti;
+
+  // verify if user exists before processing the request
+  if (!(await verifyUserExists(userId))) {
+    // User does not exist, log and return accepted response
+    console.warn(logMessages.SIGNAL_WARN_USER_NOT_FOUND, {
+      userId: userId,
+      correlationId: correlationId,
+      requestType: 'PURGE_ACCOUNT',
+    });
+    return generateResponse(StatusCodes.ACCEPTED, ReasonPhrases.ACCEPTED);
+  }
+
```

Comment:
I think the intent is that we would check if a user exists for _all_ signals - so this needs to live in the `request-handler.ts`